packages/calcite-components/src/components/tree-item/tree-item.tsx

```diff
@@ -227,17 +231,21 @@ export class TreeItem implements ConditionalSlotComponent, InteractiveComponent
 
     const checkbox =
       this.selectionMode === "ancestors" ? (
-        <label class={CSS.checkboxLabel} key="checkbox-label">
-          <calcite-checkbox
-            checked={this.selected}
-            class={CSS.checkbox}
-            data-test-id="checkbox"
-            indeterminate={this.hasChildren && this.indeterminate}
-            scale={this.scale}
-            tabIndex={-1}
-          />
-          {defaultSlotNode}
-        </label>
+        <div class={CSS.checkboxContainer}>
+          <div class={CSS.checkbox}>
```

Comment:
nitpick: do we need this div? maybe the styles could just be applied to the `calcite-icon` element?

Reply:
I removed the div, thanks for catching that!